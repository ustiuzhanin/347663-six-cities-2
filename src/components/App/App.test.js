import React from 'react';
import App from './App.jsx';
import renderer from 'react-test-renderer';

test(`App's snapshot`, () => {
  const tree = renderer.create(<App />).toJSON();

  expect(tree).toMatchSnapshot();
});
