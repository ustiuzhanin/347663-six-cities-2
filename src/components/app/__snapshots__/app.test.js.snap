// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App's snapshot 1`] = `
<Switch>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Login)",
        "type": [Function],
      }
    }
    exact={true}
    path="/login"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(OfferPage)",
        "type": [Function],
      }
    }
    exact={true}
    path="/offer/:id"
  />
</Switch>
`;
